bakery/src/tasks/disassemble-book.js

```diff
@@ -13,7 +13,7 @@ const task = () => {
       },
       inputs: [
         { name: 'book' },
-        { name: 'baked-book' },
+        { name: 'checksum-book' },
```

Comment:
This relates to my comment in `checksum-book.js`, but I think replacing this input versus adding an additional will go against the spirit of the other work to clean up the data flow in our tasks.

Reply:
I will leave the data flow like it is for now because it would be too different to the other steps.
But of course I have nothing against later optimization.